src/app/custom/translations.tsv

```diff
@@ -910,5 +910,5 @@
 "delete_all"	"Delete all"	"Tout supprimer"
 "transformer_delete_all_title"	"Delete all transformation operation"	"Supprimer toutes les opérations de transformation"
 "transformer_delete_all_description"	"Are you sure you want to delete all transformation operations ?"	"Êtes-vous sûr de vouloir supprimer toutes les opérations de transformation ?"
-"uncheck_all_value_facet"	"Uncheck all"	"Tout décocher"
-"check_all_value_facet"	"Check all"	"Tout cocher"
\ No newline at end of file
+"check_all_value_facet"	"Check all"	"Tout cocher"
+"check_all_value_facet_tooltip"	"You have more than 20 values for this facet. So we disable select all optional"	"Vous avez plus de 20 valeurs pour cette facette. Nous avons donc désactivé la sélection de toutes les valeurs"
```

Comment:
May be use a parameter in the translation to dynamically change the max facet value